src/metrics.js

```diff
@@ -104,7 +103,6 @@ export default {
   increment: name => sendFeatureMetric(name),
   getOsName,
   sendCompletionSelected,
-  version: kitePkg.version,
```

Comment:
I removed this mainly because it was causing a circular dependency when factoring out the docs commands from. It was only used in one other place, which I replaced with a direct import from `package.json`. I'm not too sure about the other fields

Reply:
It looks like `events.js` imported from `metrics.js` which imports from `package.json`. Are there other references to the package version that would create a circular dependency?

Reply:
Okay, I did some more digging since I mostly went off a hunch based off this issue https://github.com/webpack/webpack/issues/11444, which was the actual error that I was getting.

`metrics.js` imports from `kite.js`, which imports `events.js` which imports `metrics.js`. This isn't a problem since JavaScript allows for circular imports but since now `docs.ts` imports `metrics.js` and _TypeScript_ doesn't allow for circular imports, the linked error comes up (is my hypothesis, but i'm not entirely sure). 

Going off this, I can also break this circular dependency by using the util `promisifiedKiteAPIRequest` because the only reason `metrics.js` imports `kite.js` is for the `request` method (just verified this works). This seems like the correct thing to do.

It's a separate issue whether we want to expose `plugin_version` from `metrics.js`. There are other places that also import the version number from `package.json` (eg, `kite.js`). Do you have an opinion what should be the source of truth for this?

Reply:
Gotcha, thanks for the context. 

I agree with you here, transitioning to `promisifiedKiteAPIRequest` makes the most sense to me here. Importing from `package.json` wherever it's needed also sounds good, especially if refactoring out of importing `kite.js` resolves the compilation issues in question.